docs/lti-integration.md

```diff
@@ -0,0 +1,151 @@
+# LTI 1.3 Integration
+
+Code.org is working on an LTI 1.3 compliant integration for Canvas. Currently,
+we are beta testing with a small group of Canvas users, and plan to release our
+integration in 2024.
+
+The following documentation details the steps required to install Code.org as an
+LTI tool in Canvas. As an outcome of following these steps, you will supply the
```

Comment:
Completing the following steps will generate a 'Client ID' and a 'Deployment ID'. Please supply these two IDs to Code.org via [this form](https://forms.gle/hLqdbgQ1VhZdKR7N8) to complete your installation